CONTRIBUTING.md

```diff
@@ -111,16 +111,34 @@ $ meson test
 
 ## Windows support (experimental)
 
-To run on windows, install [msys2](msys2.org) and follow the instructions to set up a development environment. Once the setup is done, install the following dependencies:
+To run on windows, install [msys2](msys2.org) and follow the instructions to set up a development environment. Once the configuration is done, follow the instructions:
+
+### Install the dependencies
 
 ```bash
 $ pacman -Suy
-$ pacman -S git mingw-w64-ucrt-x86_64-meson mingw-w64-ucrt-x86_64-gtk4 mingw-w64-ucrt-x86_64-python3 mingw-w64-ucrt-x86_64-python3-gobject mingw-w64-ucrt-x86_64-libadwaita mingw-w64-ucrt-x86_64-gstreamer mingw-w64-ucrt-x86_64-gst-plugins-good mingw-w64-ucrt-x86_64-gtksourceview5 mingw-w64-ucrt-x86_64-librsvg mingw-w64-ucrt-x86_64-desktop-file-utils
+$ pacman -S git mingw-w64-ucrt-x86_64-meson mingw-w64-ucrt-x86_64-gtk4 mingw-w64-ucrt-x86_64-python3 mingw-w64-ucrt-x86_64-python3-gobject mingw-w64-ucrt-x86_64-libadwaita mingw-w64-ucrt-x86_64-gstreamer mingw-w64-ucrt-x86_64-gst-plugins-good mingw-w64-ucrt-x86_64-gtksourceview5 mingw-w64-ucrt-x86_64-librsvg mingw-w64-ucrt-x86_64-desktop-file-utils unzip meson cmake
```

Comment:
Note that meson is already in this list (as mingw-w64-ucrt-x86_64-meson). As per cmake, it's not needed here, since we're not building anything else manually that requires cmake.

Reply:
Even so, the unzip package is necessary for the subsequent steps.

Reply:
Unzip is fine, the other aren't necessary.

Reply:
A recent change we merged made pandoc not needed anymore to build Gameeky, can you update accordingly?

Reply:
@Tom5521 any chance to get this done ? :)